store/src/java-test/com/zimbra/cs/service/mail/GetFilterRulesTest.java

```diff
@@ -552,4 +552,99 @@ public void testEnvelopeTestNumericComparison() throws Exception {
 
     }
 
```

Comment:
done. pls review https://github.com/Zimbra/zm-mailbox/pull/171 for the the recommended changes